### DISCUSSION TITLE
Is this issue already taking care of ?

### DISCUSSION THREAD

**Discussion by ghost** [filamentphp/filament#7283] [General]
Hello, If you have a table with multiple columns (like 15+), and then scroll right, you'll see this ui issue:

<img width="1464" alt="Screenshot 2023-07-29 at 18 54 22" src="https://github.com/filamentphp/filament/assets/5849907/bbfb083c-cfc2-41aa-b0eb-783db430841b">

The table will get into the sidebar, and a huge black space will be left at right.

<img width="1379" alt="Screenshot 2023-07-29 at 18 56 39" src="https://github.com/filamentphp/filament/assets/5849907/9844bba0-121c-4cbd-bc66-847c639144cd">

**Comment by sumardi** [comment]
Looks like a bug. Please open an issue with a reproduction repo.

**Comment by danharrin** [comment]
Please check if #7287 solves it

**Reply by zepfietje** [reply]
No need anymore, it's fixed.

**Comment by zepfietje** [comment]
Fixed by https://github.com/filamentphp/filament/pull/7287.
